python/ray/tune/search/sample.py

```diff
@@ -526,21 +542,21 @@ def get_sampler(self):
     def sample(
         self,
         domain: Domain,
-        spec: Optional[Union[List[Dict], Dict]] = None,
+        config: Optional[Union[List[Dict], Dict]] = None,
```

Comment:
Is this type correct? Should just be Dict?

Reply:
it does work for list of dicts too though. doesn't have to be a single dict.